modules/aws-eks-teams/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.13"
+
+  required_providers {
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.7.0"
+    }
+  }
+}
```

Comment:
Can this be added to the `versions.tf` file in the root folder? It's good to have all the versions in one file.

Reply:
or we could use `provider = kubectl` in resource block

Reply:
The teams module needs to have that version as-well, since TF will try to pickup the `kubectl` provider from hashicorp, here's how it looks like when we keep it only on the root folder(note the `hashicorp/kubectl`):
```
Initializing modules...

Initializing the backend...

Initializing provider plugins...
- Reusing previous version of hashicorp/helm from the dependency lock file
- Reusing previous version of hashicorp/null from the dependency lock file
- Reusing previous version of terraform-aws-modules/http from the dependency lock file
- Reusing previous version of hashicorp/local from the dependency lock file
- Reusing previous version of hashicorp/kubernetes from the dependency lock file
- Reusing previous version of gavinbunney/kubectl from the dependency lock file
- Finding latest version of hashicorp/kubectl...
- Reusing previous version of hashicorp/cloudinit from the dependency lock file
- Reusing previous version of hashicorp/aws from the dependency lock file
- Using previously-installed gavinbunney/kubectl v1.13.1
- Using previously-installed hashicorp/cloudinit v2.2.0
- Using previously-installed hashicorp/aws v3.68.0
- Using previously-installed hashicorp/helm v2.4.1
- Using previously-installed hashicorp/null v3.1.0
- Using previously-installed terraform-aws-modules/http v2.4.1
- Using previously-installed hashicorp/local v2.1.0
- Using previously-installed hashicorp/kubernetes v2.7.1
╷
│ Error: Failed to query available provider packages
│
│ Could not retrieve the list of available versions for provider hashicorp/kubectl: provider registry
│ registry.terraform.io does not have a provider named registry.terraform.io/hashicorp/kubectl
│
│ Did you intend to use gavinbunney/kubectl? If so, you must specify that source address in each module
│ which requires that provider. To see which modules are currently depending on hashicorp/kubectl, run
│ the following command:
│     terraform providers
```
See https://github.com/gavinbunney/terraform-provider-kubectl/issues/39
https://github.com/gavinbunney/terraform-provider-kubectl/issues/91